sycl/test-e2e/Regression/commandlist/Inputs/FindPrimesSYCL.cpp

```diff
@@ -105,7 +105,7 @@ float find_prime_s(work *w) {
     }
   }
 
-  std::chrono::duration<double> diff{0};
```

Comment:
I don't think we need this to use float. This is on host so it should be fine.

Reply:
Removed.